elytron/src/test/resources/org/wildfly/extension/elytron/security-realms.xml

```diff
@@ -2,6 +2,7 @@
     <security-realms>
         <aggregate-realm name="AggregateOne" authentication-realm="RealmThree" authorization-realm="RealmFour" />
         <aggregate-realm name="AggregateTwo" authentication-realm="RealmThree" authorization-realms="AggregateOne RealmFour" />
+        <aggregate-realm name="AggregateThree" authentication-realm="RealmThree" authorization-realm="RealmFour" principal-transformer ="RegexOne"/>
```

Comment:
@ashley-abdelsayed98 : Where is this AggregateThree realm used? Is there some typo in this AggregateThree definition?

Reply:
@marekkopecky The purpose of this descriptor is predominantly to test various permutations of configuration can both be parsed and marshalled again - sometimes other definitions in the same file are cross referenced but not always.